lib/services/metadata.test.js

```diff
@@ -155,4 +155,70 @@ describe(_.startCase(filename), function () {
         });
     });
   });
+
+  describe('checkProps', function () {
+    const fn = lib[this.title];
+
+    it ('returns false for a string prop with value false', function () {
+      db.getMeta.returns(Promise.resolve({published: false}));
```

Comment:
Remember to add spacing inside an object declaration
```suggestion
      db.getMeta.returns(Promise.resolve({ published: false }));
```